docker-compose.test.yaml

```diff
@@ -5,6 +5,36 @@ version: '3.4'
 

```

Comment:
Now that the test containers are independent of the main service containers, the test script should be updated to run with only this Docker Compose file. It currently extends `docker-compose.yaml` with `docker-compose.test.yaml` so brings up the main service stack alongside the test stack.

Similar change to the test step in the build pipeline.